docker/nginx/conf.d/client.conf

```
@@ -440,7 +440,7 @@ server {
 
 		proxy_read_timeout 600;
 		proxy_set_header User-Agent: Sia-Agent;
-		# proxy this call to siad /skynet/skylink/ endpoint (make sure the ip is correct)
+		proxy_redirect $skylink_v1 $skylink_v2;
 		proxy_pass http://siad/skynet/skylink/$skylink_v1$path$is_args$args;
```

```suggestion
		proxy_pass http://siad/skynet/skylink/$skylink_v2$path$is_args$args;
```

Q: Is this not sufficient for the fix?

---

it would but this would trigger registry call on skyd because skyd would have to resolve the skylink v2 and we already have it resolved